src/main/resources/TotemCode/TotemSheet.xml

```diff
@@ -1206,200 +1180,19 @@ XWiki.widgets.TotemModalPopup = Class.create(XWiki.widgets.ModalPopup, {
-                            $.event.trigger( "ajaxSuccess", [xml, s] );
-                    } else
-                        $.handleError(s, xml, status);
-                } catch(e) 
-				{
-                    status = "error";
-                    $.handleError(s, xml, status, e);
-                }
-
-                // The request was completed
-                if( s.global )
-                    $.event.trigger( "ajaxComplete", [xml, s] );
-
-                // Handle the global AJAX counter
-                if ( s.global &amp;&amp; ! --$.active )
-                    $.event.trigger( "ajaxStop" );
-
-                // Process result
-                if ( s.complete )
-                    s.complete(xml, status);
-
-                $(io).unbind()
-
-                setTimeout(function()
-									{	try 
-										{
-											$(io).remove();
-											$(form).remove();	
-											
-										} catch(e) 
-										{
-											$.handleError(s, xml, null, e);
-										}									
-
-									}, 100)
-                xml = null
-
-            }
-        }
-        // Timeout checker
-        if ( s.timeout &gt; 0 ) 
-		{
-            setTimeout(function(){
-                // Check to see if the request is still happening
-                if( !requestDone ) uploadCallback( "timeout" );
-            }, s.timeout);
-        }
-        try 
-		{
-			var form = $('#' + formId);
-			$(form).attr('action', s.url);
-			$(form).attr('method', 'POST');
-			$(form).attr('target', frameId);
-            if(form.encoding)
-			{
-				$(form).attr('encoding', 'multipart/form-data');      			
-            }
-            else
-			{	
-				$(form).attr('enctype', 'multipart/form-data');			
-            }			
-            $(form).submit();
-
-        } catch(e) 
-		{			
-            $.handleError(s, xml, null, e);
-        }
-		$('#' + frameId).load(uploadCallback);
-        return {abort: function () {}};	
-	},
-	uploadHttpData: function( r, type ) {
-        var data = !type;
-        data = type == "xml" || data ? r.responseXML : r.responseText;
-        // If the type is "script", eval it in global context
-        if ( type == "script" )
-            $.globalEval( data );
-        // Get the JavaScript object, if JSON is used.
-        if ( type == "json" )
-            eval( "data = " + data );
-        // evaluate scripts within html
-        if ( type == "html" )
-            $("&lt;div&gt;").html(data).evalScripts();
-		return data;
-    },
-handleError: function( s, xhr, status, e ) {
-    // If a local callback was specified, fire it
-    if ( s.error ) {
-        s.error.call( s.context || window, xhr, status, e );
-    }
-
-    // Fire the global callback
-    if ( s.global ) {
-        (s.context ? jQuery(s.context) : jQuery.event).trigger( "ajaxError", [xhr, s, e] );
-    }
-}
-})
+      <code>document.observe('xwiki:dom:loading', function() {
+// Configure require.js to use jquery-form (https://github.com/malsup/form)
```

Comment:
Please put these lines in the top of the previous JSX object, and remove this useless object.